extensions/markdown-language-features/src/commands/openDocumentLink.ts

```diff
@@ -58,7 +59,7 @@ export class OpenDocumentLinkCommand implements Command {
 
 	public static async execute(engine: MarkdownEngine, args: OpenDocumentLinkArgs) {
 		const fromResource = vscode.Uri.parse('').with(args.fromResource);
-		const targetResource = vscode.Uri.parse('').with(args.path);
+		const targetResource = fromResource.scheme === 'file' && isString(args.path) ? vscode.Uri.file(args.path) : vscode.Uri.parse('').with(args.fromResource);
```

Comment:
The second part should be `vscode.Uri.parse('').with(args.path)` - keeping it the same as it is currently.

Reply:
oops missed that, thanks!